diff -Nrbu seamonkey-2.40/seamonkey-2.40/mozilla/configure seamonkey-2.40-OK/seamonkey-2.40/mozilla/configure
--- seamonkey-2.40/seamonkey-2.40/mozilla/configure	2016-01-19 05:49:57.000000000 +0300
+++ seamonkey-2.40-OK/seamonkey-2.40/mozilla/configure	2016-03-15 18:51:38.905593797 +0300
@@ -1271,7 +1271,7 @@
 MOZJPEG=62
 MOZPNG=10617
 NSPR_VERSION=4
-NSPR_MINVER=4.10.10
+NSPR_MINVER=4.10.8
 NSS_VERSION=3
 
 GLIB_VERSION=2.22
@@ -15421,7 +15421,7 @@
   echo "$ac_t""no" 1>&6
 fi
 
-	min_nss_version=3.20.1
+	min_nss_version=3.19.1
 	echo $ac_n "checking for NSS - version >= $min_nss_version""... $ac_c" 1>&6
 echo "configure:15427: checking for NSS - version >= $min_nss_version" >&5
 
